notebooks/milestone_1.ipynb

```diff
@@ -316,69 +316,69 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>1889-01-01 12:00:00</th>\n",
-       "      <td>-35.439867</td>\n",
-       "      <td>-33.574619</td>\n",
-       "      <td>141.5625</td>\n",
-       "      <td>143.4375</td>\n",
-       "      <td>4.244226e-13</td>\n",
-       "      <td>MPI-ESM-1-2-HAM</td>\n",
+       "      <td>NaN</td>\n",
```

Comment:
@zmerpez I think something went wrong with your execution, can you try re-running to see if you no longer get NaNs?